### PR TITLE
Cohesive color theme

### PR DESCRIPTION
**Pull request by bhawna-sharmaa**

### CODE REVIEW

File: frontend/src/styles/MenuViewStyles.js [hunk 1]
```diff
@@ -8,10 +8,11 @@ export const MenuViewStyles = makeStyles({
     alignItems: "center",
   },
   Backbutton: {
-    backgroundColor: "#C8BFE7",
+    marginTop: "00px",
```
Comment (NerajM) [inline frontend/src/styles/MenuViewStyles.js]: remove unnecessary line

File: frontend/src/components/TopHeader.js [hunk 2]
```diff
@@ -1,17 +1,21 @@
 import React from "react";
-import { makeStyles } from "@material-ui/core";
+import { makeStyles, Button } from "@material-ui/core";
+import { Link } from "react-router-dom";
```
Comment (NavyaGupta) [inline frontend/src/components/TopHeader.js]: Remove as its unused

File: frontend/src/styles/MenuViewStyles.js (hunk 1 above)
Comment (NavyaGupta) [inline frontend/src/styles/MenuViewStyles.js]: Remove

File: frontend/src/styles/MenuViewStyles.js [hunk 3]
```diff
@@ -8,10 +8,11 @@ export const MenuViewStyles = makeStyles({
...
+    backgroundColor: "#3f51b5",
+    color: "#FFFFFF",
```
Comment (NavyaGupta) [inline frontend/src/styles/MenuViewStyles.js]: Try changing this white

File: frontend/src/components/HostView.js [hunk 4]
```diff
@@ -36,7 +36,7 @@ export const HostView = (props) => {
   return (
     <div>
       <TopHeader />
-      <PageNameHeader title="Host Game" onClick={props.goBack}></PageNameHeader>
+      <PageNameHeader onClick={props.goBack}></PageNameHeader>
```
Comment (NavyaGupta) [inline frontend/src/components/HostView.js]: This doesnt need header as its a dialog

**Review by NavyaGupta — COMMENTED** [review]
We need to rebase. Looks good otherwise

**Pull request closed without merging**